File: Zerojudge-i510.py
# Author : ysh
# 06/27/2022 Mon  7:46:24.01
while True:
    try:
        a = input()
        b = [i for i in input().split()]
        c = b[0].find(b[1])
        if c == -1:
            print("No")
            continue
        print("Yes\npos: " + str(c))
    except:
        break
quit()

File: Zerojudge-i510.cpp
// Author : ysh
// 06/27/2022 Mon  7:50:25.96
#include<bits/stdc++.h>
using namespace std;
int main() {
    ios::sync_with_stdio(false);
    cin.tie(0);

    string a;
    while(cin>>a>>a>>a) {
        string b;cin>>b;
        int c = a.find(b);
        if(c == -1) {
            cout<<"No\n";
            continue;
        }
        cout<<"Yes\npos: "<<c<<"\n";
    }
    return 0;
}